app/views/nonprofits/show.html.erb

```diff
@@ -6,6 +6,24 @@
   <% end %>
 </h1>
 
+<% unless @nonprofit.social_profile.nil? %>
+  <% if @nonprofit.social_profile.facebook.present? %>
```

Comment:
@ianroberts131 In `<% if @nonprofit.has_profile(facebook) %>` there is a variable named `facebook` as the argument to `has_profile(...)`.

The error is due to the `facebook` variable not yet existing. To make it exist, we would assign it a value before using it:

```
<% facebook = ...some value... %>
The facebook variable exists, now it can be used:
<% if @nonprofit.has_profile(facebook) %>
```

Lets talk this one through further today on the call.